ui/src/components/Rules/RuleGroup/index.tsx

```diff
@@ -97,7 +102,16 @@ const RuleGroup = (props: {
             </div>
           </div>
         </div>
-        <div className="grid w-full grid-cols-1 xl:grid-cols-2">
+        <div className="grid w-full grid-cols-1 xl:grid-cols-3">
+          <div>
+            <RunButton
+              onClick={() => props.onExecute(props.group)}
+              text="Run"
+              executing={props.isExecuting}
```

Comment:
I think this is supposed to make the run button show a spinner like the top level Run Rules button when executing? It's not appearing for me though.